app/src/androidTest/java/com/sdp/movemeet/utilityTest/ActivitiesUpdaterTest.java

```diff
@@ -1,30 +1,16 @@
-package com.sdp.movemeet.utility;
+package com.sdp.movemeet.utilityTest;
```

Comment:
We can remove the test in the package

Reply:
In the package name, right! As said by Dorian this morning. Jean-Luc is doing it at the moment!